.github/workflows/deploy_checks_lambda.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy Checks Updater Lambda
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/deploy_lambda.yml'
+      - 'fbossci-aws-lambas/us-east-1/checks-cron/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Deploy
+        uses: appleboy/lambda-action@1e05c1377056f21ebb2ce69b910bc16b943c2a66
```

Comment:
qq: `timeout` defaulted to 3 sec according to https://github.com/appleboy/lambda-action#input-variables
did we profile the speed of this lambda

Reply:
I ended up setting it manually to 15 seconds on the UI so added that here